src/backend/routes/tankRoutes.js

```diff
@@ -116,8 +116,6 @@ router.get('/getBotTanks', tankController.getBotTanks);
 // Route call: /getTankById/<tankId>
 // Body: N/A
 // returns the tank object
-router.get('/getTankById/:tankId', [
-	check('tankId', 'a valid MongoId is required.').isMongoId()
-], tankController.getTankById);
+router.get('/getTanksById', tankController.getTankById);
```

Comment:
Seeing that there is a use of the query parameters, should you not use the express validator here? What is expected to go here? It seems like its supposed to be taking something in. Otherwise you can give this thing random stuff. Also it seems that this has been edited to the point that the comment to describe this route should be updated.